data/example-data-oracle-book/src/test/java/io/americanexpress/data/oracle/book/test/config/OracleDataConfigTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.americanexpress.data.oracle.book.test.config;
+
+import com.zaxxer.hikari.HikariDataSource;
+import io.americanexpress.synapse.data.oracle.config.BaseOracleDataConfig;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.test.context.ActiveProfiles;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+/**
+ * {code OracleDataConfigTest} Configuration file uses h2 database for integration test purpose.
+ */
+@TestConfiguration
+@PropertySource("classpath:data-oracle-book-application-test.properties")
+@EnableAutoConfiguration
+@ActiveProfiles("test")
+public class OracleDataConfigTest extends BaseOracleDataConfig {
```

Comment:
should this be OracleDataTestConfig and just import the DataBookConfig.class ?

Reply:
I can do that